src/main/java/mate/academy/onlinebookstore/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.onlinebookstore.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.Size;
+import mate.academy.onlinebookstore.validation.FieldMatch;
+
+@FieldMatch(field = "password",
+        fieldMatch = "repeatPassword",
+        message = "The passwords do not match."
+)
+public record UserRegistrationRequestDto(
+        @Email(message = "Invalid format email.")
+        String email,
+
+        @NotNull(message = "Invalid password. Password shouldn't be null.")
```

Comment:
Use `@NotBlank `for String

```suggestion
        @NotBlank(message = "Invalid password. Password shouldn't be null.")
```